src/renderer/controllers/prefs-controller.js

```diff
@@ -17,7 +17,7 @@ module.exports = class PrefsController {
       setup: function (cb) {
         // initialize preferences
         state.window.title = 'Preferences'
-        state.unsaved = Object.assign(state.unsaved || {}, {prefs: state.saved.prefs || {}})
```

Comment:
yeah, there was an immutability problem,
the saved and unsaved perfs object was the same one.
Fixed :smile: